modules/porous_flow/include/materials/PorousFlowHystereticRelativePermeabilityLiquid.h

```diff
@@ -0,0 +1,47 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "PorousFlowHystereticRelativePermeabilityBase.h"
+
+/**
+ * Material to compute liquid relative permeability for 1-phase and 2-phase hysteretic models
+ */
+class PorousFlowHystereticRelativePermeabilityLiquid
+  : public PorousFlowHystereticRelativePermeabilityBase
+{
+public:
+  static InputParameters validParams();
+
+  PorousFlowHystereticRelativePermeabilityLiquid(const InputParameters & parameters);
+
+protected:
+  /// Phase number of liquid phase
+  const unsigned _liquid_phase;
```

Comment:
Could you put `_liquid_phase` into the base class here and remove it from the gas and liquid classes?

Reply:
bummer that i can't do this without being rather obscure (in my opinion)